bokehjs/src/lib/models/tools/toolbar.ts

```diff
@@ -69,7 +69,9 @@ export class Toolbar extends ToolbarBase {
 
   connect_signals(): void {
     super.connect_signals()
-    this.connect(this.properties.tools.change, () => this._init_tools())
+
+    const {tools, active_drag, active_inspect, active_scroll, active_tap, active_multi} = this.properties
+    this.on_change([tools, active_drag, active_inspect, active_scroll, active_tap, active_multi], () => this._init_tools())
   }
```

Comment:
With this code, what happens when there's a proxy toolbar?

Reply:
`ProxyToolbar` doesn't support `active_` properties. Additionally, `ProxyTool` didn't listen to sub-tool `active` changes, so at least I fixed that, so now it's possible to activate a proxy by activating a single tool.